tests/helpers.bash

```diff
@@ -148,7 +155,7 @@ function run_buildah() {
 
         # stdout is only emitted upon error; this echo is to help a debugger
         echo "\$ $BUILDAH_BINARY $*"
-        run timeout --foreground --kill=10 $BUILDAH_TIMEOUT ${BUILDAH_BINARY} ${REGISTRY_OPTS} ${ROOTDIR_OPTS} "$@"
+        run timeout --foreground --kill=10 $BUILDAH_TIMEOUT ${BUILDAH_BINARY} ${BUILDAH_REGISTRY_OPTS} ${ROOTDIR_OPTS} "$@"
```

Comment:
Thinking ahead: some day this will be vendored into podman, which will then run the buildah-bud-under-podman tests. How do you foresee making those podman tests work with this new scheme? Do you think podman needs a (hidden) `--registries-conf-dir` option? Will some larger subset of tests need to be skipped now?

Reply:
Since Buildah and (soon) Podman both share `common/libimage` we could use an env variable rather than a CLI flag. Unless explicitly set [libimage](https://github.com/containers/common/blob/master/libimage/runtime.go#L27) will look for the CONTAINERS_REGISTRIES_CONF and REGISTRIES_CONFIG_PATH.

Reply:
Sorry, I didn't check the code in question.  In this specific case, libimage isn't executed.

Reply:
Yes I think we add the hidden option to Podman.